components/drawer/overview.md

```diff
@@ -12,12 +12,19 @@ position: 0
 
 The <a href="https://www.telerik.com/blazor-ui/drawer" target="_blank">Blazor Drawer component</a> is a dismissible or permanently visible panel for [navigating]({%slug drawer-navigation%}) in responsive web applications. It allows switching the content of different sections of the page. You can customize its [templates]({%slug drawer-templates%}), [display]({%slug drawer-modes%}), behavior when [minimizing]({%slug drawer-mini-mode%}) and respond to [events]({%slug drawer-events%}).
```

Comment:
```suggestion
The <a href="https://www.telerik.com/blazor-ui/drawer" target="_blank">Blazor Drawer component</a> is an interactive side panel for [navigating]({%slug drawer-navigation%}) in responsive web applications. It can be always visible, or expanded and collapsed. The Drawer allows switching the content of different sections on the page. You can customize its [templates]({%slug drawer-templates%}), [expand modes]({%slug drawer-modes%}), [minimize behavior]({%slug drawer-mini-mode%}) and also respond to [events]({%slug drawer-events%}).
```